suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml

```diff
@@ -130,35 +130,51 @@ tests:
                       - node1
                       - node2
                       - node3
+                  extra_container_args:
```

Comment:
would suggest to move this test cases it to new test suite file( like `test-container-cli-args`) without updating old apply-spec test cases.

Reply:
this would enable us to run new features with every build and there would be no problem with polarion Id's association.

Reply:
Done.